CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Changed
 - civis.io functions use the "hidden" API option to keep jobs out of the UI. Deprecate the "archive" parameter in favor of "hidden".
 - civis.io.query_civis now has a "hidden" parameter which defaults to True
+- expose `polling` and `polling_args` as public attribute in PollableResults
```

Comment:
"`poller` and `poller_args`"?
"attribute" --> "attributes"
"PollableResults" --> "PollableResult"
Backticks around "PollableResult"?